### ISSUE TITLE
Top bar doesn't respond properly on window resize

### ISSUE THREAD

**Issue by rickmcgeer** [LivelyKernel/lively.next#318]
See the attached screenshot.   Not only are the icons sitting on top of each other, the mouse events are on the wrong icon.  For example, when I clicked on hand for interaction mode, I got a popup over my profile picture, as if I'd clicked on that.
![image](https://user-images.githubusercontent.com/656259/131260184-109a0ce6-1e69-4ea6-b722-f16bed8cc6fb.png)

**Comment by merryman** [comment]
This will be fixed after #320 is merged into master.

**Comment by merryman** [comment]
This is fixed now.
